AI/scripts/download_weights.py

```
@@ -0,0 +1,190 @@
+        "slug"      : f"{KAGGLE_USERNAME}/tcn-training",
+        "dst_dir"   : os.path.join(project_root, "AI/data/weights/tcn"),
+        "keep_files": ["model.pt", "scaler.pkl", "metadata.json"],
+    },
+]
+
+
+def list_output_files(slug: str) -> list:
+    """Kaggle API로 노트북 output 파일 목록 조회"""
+    owner, kernel = slug.split("/")
+    url = f"https://www.kaggle.com/api/v1/kernels/output/{owner}/{kernel}?page_token=START"
+    resp = requests.get(url, auth=(KAGGLE_USERNAME, KAGGLE_KEY))
```

Ruff (0.15.7)

[error] Probable use of `requests` call without timeout

(S113)